exercises/rotational-cipher/src/example/java/RotationalCipher.java

```diff
@@ -0,0 +1,29 @@
+public class RotationalCipher {
+    public String rotate(String data, int  shiftKey){
```

Comment:
I used just a plain text editor and command line when I first started learning java. I think it's quite nice to start with that because then you have to really understand everything you're doing and don't just have a magic idea which does lots of random stuff for you that you don't understand. But as I started doing bigger things, using an IDE is just so much nicer and quicker. So switching to an IDE is a good idea, but I think it's great that you've tried using just a text editor and command line :)